tests/cxx/quoted_includes_first.h

```diff
@@ -0,0 +1,38 @@
+//===--- quoted_includes_first.cc - test input file for iwyu --------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <iostream>
+#include <list>
+#include <map>
+#include <memory>
+#include "subfolder/indirect_subfolder.h"
+
+class quoted_includes_class {
```

Comment:
Use PascalCase for class names. That said, I'm not sure the class adds anything here, the method might as well be a free function (?)

Reply:
Removed class.